code/interfaces/CronTaskEditable.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * By implementing this interface you can define this task as editable
+ * and can implement needed logic who can edit it
+ *
+ */
+interface CronTaskEditable {
```

Comment:
Would it be better if this extended `CronTask` rather than having to implement 2 interfaces?